### ISSUE TITLE
Set Actual Quality

### ISSUE THREAD

**Issue by ghost** [cmcnamara87/ytpodcaster#3]
Thanks for making this! The SD, HD, and Full HD quality options aren't exact enough for my purposes though. It would be nice if YTPodcaster could fetch the actual quality sizes from the YouTube channel, and then present a popup with options such as 360p, 480p, 720p, 1080p, etc.

**Comment by cmcnamara87** [comment]
Yep that shouldn't be too hard. The Youtube wiki page contains a list of all the quality / codecs on Youtube - http://en.wikipedia.org/wiki/YouTube#Quality_and_codecs.  Although now that I'm looking it seems that 37 (1080p) has been discontinued, I'll have to look into that (see #4). The other option is letting you download non-mp4 videos (see #5). They won't be able to be synced to apple devices, but you could still potentially download them with iTunes.

**Comment by ghost** [comment]
That's good news. :smile:

BTW, only reason I want to download them to iTunes is so I can sync with Apple devices.

**Comment by cmcnamara87** [comment]
Ah. Then we are stuck with the MP4s. I'll actually see if any of the apple devices can play non-mp4 h264 videos. Not sure if its tied to the container as well as the codec.
